includes/iot-edge-verify-iot-edge-continuous-integration-continuous-deployment.md

```diff
@@ -30,4 +30,8 @@ To trigger a build job, you can either push a commit to source code repository o
 
 6. Click **dev** stage to see release logs.
 
-    ![Release logs](./media/iot-edge-verify-iot-edge-continuous-integration-continuous-deployment/release-logs.png)
\ No newline at end of file
+    ![Release logs](./media/iot-edge-verify-iot-edge-continuous-integration-continuous-deployment/release-logs.png)
+
+7. I need more help. My pipeline is failing.
```

Comment:
I would delete this line, and just keep the content on line 37. You can update the first sentence with something like, "If your pipeline is failing, start by looking at the logs..."